client/boot/app.js

```diff
@@ -1,5 +1,6 @@
 // Initialize polyfills before any dependencies are loaded
 import './polyfills';
+import './dependencies';
```

Comment:
I would do this only in the specific section that needs the `window.wp` bindings. It's most likely the `sidebar-woo` component, async-loaded as a separate chunk?

Reply:
Good suggestion.  I'm hoping for a more elegant solution in the final PR but if we follow this pattern, only registering dependencies on the global when needed sounds good.